src/license.rs

```diff
@@ -0,0 +1,38 @@
+use std::fs;
+use std::io::{BufRead, BufReader};
+use std::path::Path;
+
+use crate::types::CheckResult;
+
+fn check_if_license_is_present(path: &Path) -> CheckResult<bool> {
+    let f =
+        fs::File::open(path).map_err(|e| format!("Failed to open `{}`: {}", path.display(), e))?;
+    let f = BufReader::new(f);
+    let header: Vec<String> = f.lines().take(3).map(|x| x.unwrap()).collect();
+    Ok(header.len() == 3 &&
+        header[0].starts_with("// Copyright ") &&
+        header[0].ends_with(", The Gtk-rs Project Developers.") &&
```

Comment:
That wouldn't allow me to run it on gstreamer-rs, and would be a bit strict anyway.

Also note that "The Gtk-rs Project Developers" as a copyright declaration is rather useless as that's not a legal entity. That's as useful as writing "Santa Clause" there :)

See also https://github.com/gtk-rs/gtk-rs/issues/60 . If we fix this now let's do it properly instead of adding more useless information everywhere.